<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
   background-color: rgb(255, 255, 204);
   color: #cccccc;
}
QComboBox {
	background-color:rgb(153, 204, 255);
}
QPushButton {
    background-color: rgb(34, 185, 255);
    border-width: 2px;
    border-color: rgb(117, 156, 195);
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
	alternate-background-color: rgb(220, 220, 220);
}

QPushButton:hover {
   background-color: rgb(122, 163, 204);
}
QTreeView {
   background-color: #666666;
   color: #cccccc;
   alternate-background-color: #333333;
} 
QListView {
   background-color: #333333;
   color: #cccccc;
}
QTextEdit {
   color: rgb(0, 0, 0);
}
QScrollBar:horizontal {
   height: 20px;
}
QScrollBar:vertical {
   width: 20px;
}
QTabWidget::pane {
   border-top: 2px solid #cccccc;
}
QLabel {
   color: rgb(102, 102, 102); 
}

QCheckBox {
   color: white; 
}

QRadioButton{
   color:white;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Droid Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Droid Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Droid Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_Name">
        <property name="font">
         <font>
          <family>Droid Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Tel">
        <property name="font">
         <font>
          <family>Droid Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Address">
        <property name="font">
         <font>
          <family>Droid Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
